### PR TITLE
[FR] Update "Ranking criteria"

### PR DESCRIPTION
**Pull request by DoryanR**
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/fr.md [hunk 1]
```diff
@@ -297,3 +293,8 @@ Cette catégorie contient des déclarations d'allocation explicites de concepts
...
   - **Lorsque vous utilisez plusieurs commandes du même type sur un sprite, essayez de laisser au moins 16 ms entre leurs temps de démarrage.** 60 commandes par seconde sont souvent plus que suffisantes pour permettre à un sprite d'effectuer des transitions fluides dans une configuration moyenne. Ceci dans le but de réduire la taille des fichiers et les temps de chargement.
   - **Suppression des sprites activés par les déclencheurs après utilisation.** Les déclencheurs s'activent dès leur première commande possible et restent actifs jusqu'à la fin de la difficulté, c'est pourquoi il est préférable de les faire disparaître lorsqu'ils sont terminés.
+
+## Notes
+
+[^personnage-voix-acteur]: CV (Character Voice) est utilisé lorsque la musique est chantée dans le personnage par l'acteur de la voix du personnage. `VO` (Voice Over) est utilisé quand une musique est chantée dans le personnage par quelqu'un d'autre que l'acteur de la voix du personnage.
+[^dimensions-maximales]: Les arrière-plans qui sont également utilisés comme images de storyboard peuvent suivre [la règle des images de storyboard pour les dimensions maximales](/wiki/Ranking_Criteria#règles.7) à la place.
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/fr.md:300]:
```suggestion
[^dimensions-maximales]: Les arrière-plans qui sont également utilisés comme images de storyboard peuvent suivre [la règle des images de storyboard pour les dimensions maximales](/wiki/Ranking_Criteria#règles.7) à la place.

```

File: wiki/Ranking_Criteria/fr.md [hunk 2]
```diff
@@ -121,7 +117,7 @@ Il est important de noter avant de les utiliser :
...
 - **Si la musique d'une beatmap est composée de deux ou plusieurs musiques, indiquez clairement les titres des musiques avec un symbole de séparation entre elles ou utilisez un titre descriptif de son contenu.** Si le titre devient trop long, un titre descriptif doit être utilisé à la place.
 - **Si un symbole est utilisé pour regrouper des parties d'un titre, un espace doit être utilisé avant et après le groupe, mais pas directement avant ou après les symboles à l'intérieur des groupes.**
-- **Si une série applicable comme source contient des sous-séries, le tag le plus précis doit être utilisé comme source.** Si une musique a plusieurs sous-séries applicables, la série/franchise principale peut être utilisée à la place.
+- **Si le média applicable au champ source contient des sous-séries, l'étiquette la plus précise doit être utilisée comme source.** Si une musique a plusieurs sous-séries applicables, la source principale du média peut être utilisée à la place.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/fr.md:120]:
```suggestion
- **Si le média applicable au champ source contient des sous-séries, le label le plus précis doit être utilisé comme source.** Si une musique a plusieurs sous-séries applicables, la source principale du média peut être utilisée à la place.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Comment by cl8n** [comment]
seems missing changes from https://github.com/ppy/osu-wiki/pull/7503/files#diff-d83e7a1fb8077f937a9a91827c6cb673767a7ebb721e3482bdc146a80802b3d2 (even though this is in the tree)

also, under L128 there are sub-bullets in EN that are missing here. not related to your changes but I think that should be updated before removing `outdated` tag

File: wiki/Ranking_Criteria/fr.md [hunk 3]
```diff
@@ -297,3 +292,8 @@ Cette catégorie contient des déclarations d'allocation explicites de concepts
...
   - **Lorsque vous utilisez plusieurs commandes du même type sur un sprite, essayez de laisser au moins 16 ms entre leurs temps de démarrage.** 60 commandes par seconde sont souvent plus que suffisantes pour permettre à un sprite d'effectuer des transitions fluides dans une configuration moyenne. Ceci dans le but de réduire la taille des fichiers et les temps de chargement.
   - **Suppression des sprites activés par les déclencheurs après utilisation.** Les déclencheurs s'activent dès leur première commande possible et restent actifs jusqu'à la fin de la difficulté, c'est pourquoi il est préférable de les faire disparaître lorsqu'ils sont terminés.
+
+## Notes
+
+[^personnage-voix-acteur]: CV (Character Voice) est utilisé lorsque la musique est chantée dans le personnage par l'acteur de la voix du personnage. `VO` (Voice Over) est utilisé quand une musique est chantée dans le personnage par quelqu'un d'autre que l'acteur de la voix du personnage.
+[^dimensions-maximales]: Les arrière-plans qui sont également utilisés comme images de storyboard peuvent suivre [la règle des images de storyboard pour les dimensions maximales](/wiki/Ranking_Criteria#règles.7) à la place.
```
Comment (cl8n) [inline wiki/Ranking_Criteria/fr.md:299]: also, the identifier here (e.g. `[^personnage-voix-acteur]`) should *not* be translated, we'll prob add that to ASC soon

File: wiki/Ranking_Criteria/fr.md [hunk 4]
```diff
@@ -130,6 +126,9 @@ Il est important de noter avant de les utiliser :
 - **Lorsqu'une musique utilise des mots répétés dans le titre ou dans le nom de l'artiste, dont l'un est en unicode et l'autre en romanisation de base, le champ romanisé doit utiliser uniquement la romanisation fournie et supprimer le mot en double.**
 - **Les trémas doivent être romanisés en équivalents de deux lettres : `ü` en `ue`, `ö` en `oe`, `ä` en `ae` et `ß` en `ss`.**
 - **Les musiques dont les métadonnées sont en russe/cyrillique doivent être romanisées en utilisant la méthode du [système BGN/PCGN](https://fr.wikipedia.org/wiki/Romanisation_BGN/PCGN) dans les champs romanisés.** Il en va de même pour le champ Source si le mappeur préfère une source romanisée.
+  - Е et е doivent être romanisés en `ye` s'ils sont seuls ou après `a`, `e`, `ё`, `и`, `о`, `у`, `ы`, `э`, `ю`, `я`, `й`, `ъ`, `ь`. Dans les autres cas, il doit être romanisé en `e`.
+  - `ё` doit être romanisé en `yo`, cependant, utilisez `o` si le caractère vient après `ж`, `ч`, `ш`, ou `щ``.
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/fr.md:130]:
```suggestion
  - `ё` doit être romanisé en `yo`, cependant, utilisez `o` si le caractère vient après `ж`, `ч`, `ш`, ou `щ`.
```

**Review by Biokax — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**